tests/navigation/test_navigation.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+
+@pytest.mark.usefixtures("risk_accepted")
+def test_network_switcher(vega: VegaService, page: Page):
+    page.goto(f"http://localhost:{vega.console_port}/#/disclaimer")
```

Comment:
Could potential move this setup into a function as it's reused across all of the tests.